step-templates/file-system-create-file.json

```diff
@@ -1,53 +1,59 @@
 {
-  "Id": "175a91a9-562e-49b9-bfa6-609a4e16bc56",
+  "Id": "ce22912c-76c7-4637-b09b-a04f8deff808",
```

Comment:
The `Id` of existing step templates should not be updated as it breaks the sync with Octopus.